scripts/database.js

```diff
@@ -337,20 +334,18 @@ export class Database {
             const vout = tx.vout.map(
                 (x) =>
                     new CTxOut({
-                        outpoint: new COutpoint({
-                            txid: x.outpoint.txid,
-                            n: x.outpoint.n,
-                        }),
                         script: x.script,
                         value: x.value,
                     })
             );
             return new Transaction({
```

Comment:
at this point why don't we just save/load the transaction hex in the db?

Reply:
We can potentially query the db to avoid loading all txs in memory as a potential future optimization if we store them in the object format like we're doing now

Reply:
having the hex saved would be nice so we would not have to change DB version anymore (for txdb at least) but if you don't like ok